src/Hyperledger.Aries.AspNetCore/Hyperledger.Aries.AspNetCore.csproj

```diff
@@ -1,5 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
This is a packable project, should target netstandard.

Reply:
fluent validation 9.x.x does not work with netstandard anymore so you can not upgrade to ver 9.x.x.

Reply:
It is used in this file MvcBuilderExtensions.cs.